rustls/src/msgs/base.rs

```diff
@@ -135,10 +135,6 @@ impl PayloadU8 {
     pub(crate) fn empty() -> Self {
         Self(Vec::new())
     }
-
-    pub(crate) fn into_inner(self) -> Vec<u8> {
-        self.0
-    }
```

Comment:
It might be nice to squash the unused fixes into the commit that reduces the internal exports before merging but I don't feel super strongly about it.